scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -289,7 +289,7 @@ namespace <%= namespace %> {
             });
         }
 
-        private void CheckoutWithTokenizedPayment(TokenizedPaymentInput tokenizedPaymentInput, CompletionCallback callback) {
+        public void CheckoutWithTokenizedPayment(TokenizedPaymentInput tokenizedPaymentInput, CompletionCallback callback) {
```

Comment:
We should an audit at some point of what we make public on Cart because it might be confusing for developer to see a method like this and use it. If I see correctly this is only made public so it can be accessed by `iOSNativeCheckout` right?